tsdb/index/inmem/meta.go

```diff
@@ -1164,33 +1164,43 @@ func (a Measurements) Union(other Measurements) Measurements {
 	return result
 }
 
-// Series belong to a Measurement and represent unique time series in a database.
-type Series struct {
-	mu          sync.RWMutex
-	Key         string
-	tags        models.Tags
+// series belong to a Measurement and represent unique time series in a database.
+type series struct {
+	// immutable
 	ID          uint64
-	measurement *Measurement
-	shardIDs    map[uint64]struct{} // shards that have this series defined
-	deleted     bool
+	Measurement *measurement
+	Key         string
+	Tags        models.Tags
+
+	mu       sync.RWMutex
+	shardIDs map[uint64]struct{} // shards that have this series defined
+	deleted  bool
```

Comment:
Can we move `deleted` down to the bottom? There will be 7 bytes of padding after `deleted`, and if it gets left here then we'll end up with a larger than necessary struct size if we add anything that's less than 8 bytes underneath.